projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cpq-quote/cpq-quote.ts

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
MAJOR: please re-structure this filr
1. remove unwanted/commented piece of code
2. try to re-use existing helper function as much as possible
3. is the exported constants used anywhere? if not remove them
4. is all the selectors used? please remove if not used.